view/frontend/web/insights.js

```diff
@@ -0,0 +1,250 @@
+                    var eventData = self.buildEventData(
+                        'Clicked', suggestion.objectID, suggestion.__indexName, suggestion.__position, suggestion.__queryID
+                    );
+                    self.trackClick(eventData);
+                });
+            });
+
+
+            if (this.config.ccAnalytics.enabled || this.config.personalization.enabled) {
+
+                $(document).on('click', this.config.ccAnalytics.ISSelector, function() {
+                    var $this = $(this);
+                    var eventData = self.buildEventData(
+                        'Clicked', $this.data('objectid'), $this.data('indexname'), $this.data('position'), $this.data('queryid')
+                    );
+
+                    self.trackClick(eventData);
+                });
+
+            }
+
+            if (this.config.personalization.enabled) {
+
+                // Clicked Events
+                var clickEvents = Object.keys(this.config.personalization.clickedEvents);
+
+                for (var i = 0; i < clickEvents.length; i++) {
+                    var clickEvent = this.config.personalization.clickedEvents[clickEvents[i]];
+                    if (clickEvent.enabled && clickEvent.method == 'clickedObjectIDs') {
+                        $(document).on('click', clickEvent.selector, function(e) {
+                            var $this = $(this);
+                            var event = self.getClickedEventBySelector(e.handleObj.selector);
+
+                            var eventData = self.buildEventData(
+                                event.eventName,
+                                $this.data('objectid'),
+                                $this.data('indexname') ? $this.data('indexname') : self.defaultIndexName
+                            );
+
+                            self.trackClick(eventData);
+                        });
+                    }
+                }
+
+                // Filter Clicked
+                if (this.config.personalization.filterClicked.enabled) {
+                    var facets = this.config.facets;
+                    var containers = [];
+                    for (var i = 0; i < facets.length; i++) {
+                        var elem = createISWidgetContainer(facets[i].attribute);
+                        containers.push('.' + elem.className);
+                    }
+
+                    algolia.registerHook('afterInstantsearchStart', function (search, algoliaBundle) {
+                        var selectors = document.querySelectorAll(containers.join(', '));
+                        selectors.forEach(function (e) {
+                            e.addEventListener('click', function (event) {
+                                var attribute = this.dataset.attr;
+                                var elem = event.target;
+                                if (elem.matches("input[type=checkbox]") && elem.checked) {
```

Comment:
`matches` will break on IE 9: see https://caniuse.com/#feat=matchesselector
why not use jquery like you did earlier with clickedObjectIDs ? 

jQuery has an API that you can use cross browser: https://api.jquery.com/is/

Reply:
Converted it use : `$(elem).is("input[type=checkbox]")`